content/docs/user-guide/networking/multiplayer/autocomponents.md

```diff
@@ -107,3 +107,34 @@ set(FILES
     Source/AutoGen/MySimpleNetPlayerComponent.AutoComponent.xml    
 )    
 ```
+
+## Network inputs
+
+Multiplayer auto-components can have *network inputs*, which are used to send input data from the player to the authoritative server. Network inputs are like special RPCs that the Multiplayer Gem creates, processes, and records at each frame of the network tick. When the authority receives a network input from the player, it stamps the network input with a frame number. Noting the frame number allows the authority to rewind, meaning it returns all the rewindable network properties to that moment in history, before it processes the input. Rewind is crucial to keep the multiplayer simulation in sync.
+
+### Attributes
+
+| Property | Description |
+| --- | --- |
+| **Name** | The name of the input. This name is used when writing or reading the input in code. |
+| **Type** | The data type that the input is stored as. |
+| **Init** | The initial value of the data. |
+
+### Example
+
+```xml
+<NetworkInput Type="StickAxis" Name="ForwardAxis" Init="0.0f" />
+<NetworkInput Type="bool"      Name="Crouch"      Init="false" />
+<NetworkInput Type="uint8_t"   Name="ResetCount"  Init="0" />
+```
+
+For a complete example, refer to [`NetworkPlayerMovementComponent.AutoComponent.xml`](https://github.com/o3de/o3de-multiplayersample/blob/development/Gem/Code/Source/AutoGen/NetworkPlayerMovementComponent.AutoComponent.xml#L21-L28).
+
+### Using network inputs in game logic
+
+In C++ and scripting, an auto-component with a network input overrides the following controllers:
+
+- `CreateInput`: The multiplayer system automatically calls `CreateInput` for the autonomous player at every network tick.
```

Comment:
I'd like something that mentions CreateInput being where devs return a net-input based on all the device input that happened between the last and current network tick. It's an important concept, because unlike single player, we cannot act immediately upon receiving keyboard input, instead we track all the device input it between ticks, and then stuff whatever happened into the net-input when asked (ie: CreateInput)

_The multiplayer system automatically calls `CreateInput` for the autonomous player at every network tick. CreateInput allows developers to return a filled out network input based on the recorded mouse/keyboard/etc occurring between each network tick._